### PR TITLE
Admin send letter borrower

### PR DESCRIPTION
**Pull request by annq3012**
## Description
reminder button and remind letter content
## Reference Requirements
- Product Backlog ID: PBI-3
- Task ID: TI-3.1

## Reference Isuees
#{issue_id}

## Migrations
YES

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: app/Http/Controllers/Admin/SendMailController.php [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Model\User;
+use Illuminate\Http\Request;
```
Comment (at-anhngo) [inline app/Http/Controllers/Admin/SendMailController.php]: cái này không xài thì xóa đi

**Review by ghost — CHANGES_REQUESTED** [review]
Check my comment

File: routes/web.php [hunk 2]
```diff
@@ -30,3 +30,4 @@
 Route::group(['prefix' => 'api', 'namespace' => 'Api'], function () {
     Route::put('users/{id}/roles', 'UserController@updateRole')->middleware('TeamSA');
 });
+Route::get('mail', 'Admin\SendMailController@sendMail')->name('sendMail');
```
Comment (at-anhngo) [inline routes/web.php]: route này không có middleware thì ai cũng có thể vào được

**Review by ghost — CHANGES_REQUESTED** [review]
Check my comment

**Review by ghost — APPROVED** [review]
LGTM

**Comment by at-huydq** [comment]
good job, LGTM.

**Pull request merged**